documentation/api/development.md

```diff
@@ -4,87 +4,56 @@ This application is dockerized. Take a look at [Dockerfile](../../api/Dockerfile
-2. Run `make setup-local` to install dependencies
-3. Run `make init start` to build the image and start the container.
-4. Navigate to `localhost:8080/docs` to access the Swagger UI.
-5. Run `make run-logs` to see the logs of the running API container
-6. Run `make stop` when you are done to delete the container.
+Run `make db-seed-local && populate-search-opportunities` to create local data in the database and make it available in the API.
 
-## Some Useful Commands
+### API Authenticaion
 
-`make test` will run all of the tests. Additional arguments can be passed to this command which will be passed to pytest like so: `make test args="tests/api/route -v"` which would run all tests in the route folder with verbosity increased. See the [Pytest Docs](https://docs.pytest.org/en/7.1.x/reference/reference.html#command-line-flags) for more details on CLI flags you can set.
-
-`make clean-volumes` will spin down the docker containers + delete the volumes.
-
-`make volume-recreate` Deletes the volumes and then re-initializes the persistant portions of the stack. This can be useful to reset your DB, or fix any bad states your local environment may have gotten into.
-
-See the [Makefile](../../api/Makefile) for a full list of commands you can run.
-
-The `make console` command initializes a Python REPL environment pre-configured with database connectivity. This allows developers to perform database queries, utilize factories for data generation, and interact with the application's models directly.
-
-- Writing a query: `dbs.query(Opportunity).all()`
-- Saving some factory generated data to the db: `f.OpportunityFactory.create()`
-
-## Docker and Native Development
-
-Several components like tests, linting, and scripts can be run either inside of the Docker container, or outside on your native machine.
-Running in Docker is the default, but on some machines like the M1 Mac, running natively may be desirable for performance reasons.
-
-You can switch which way many of these components are run by setting the `PY_RUN_APPROACH` env variable in your terminal.
-
-- `export PY_RUN_APPROACH=local` will run these components natively
-- `export PY_RUN_APPROACH=docker` will run these within Docker
+This API uses a very simple [ApiKey authentication approach](https://apiflask.com/authentication/#use-external-authentication-library) which requires the caller to provide a static key. This is specified with the `API_AUTH_TOKEN` environment variable.
 
-Note that even with the native mode, many components like the DB and API will only ever run in Docker, and you should always make sure that any implementations work within docker.
+### User Authentication
 
-Running in the native/local approach may require additional packages to be installed on your machine to get working.
+Run `make setup-env-override-file` to create the `override.env` file which will include the necessary JWT keys for running the user.
```

Comment:
It isn't super clear to me what `running the user` means here. What about something like `running user authentication within the app`?

Reply:
Updated, thanks.